docs/docs/reference/builtin-functions/feel-built-in-functions-temporal.md

```diff
@@ -98,3 +98,16 @@ abs(duration("PT5H"))
 abs(duration("-P2M"))
 // duration("P2M")
 ```
+
+## getLastDayOfTheMonth()
+
+Returns the last day of the month value of a date.
+
+- parameters:
+  - `n`: date
+- result: date
+
+```js
+getLastDayOfTheMonth(date("2022-10-1"))
+// date("2022-10-31"))
+```
```

Comment:
:x: We should align the name of the function and the parameter with the implementation.

In the implementation, the function is named `last day of the month` (but should be `last day of month`). The parameter has the name `date` instead of `n`.